BUILD.md

```diff
@@ -692,3 +693,12 @@ To run the loader test script, change to the `build/tests` directory in your
 Vulkan-Loader repository, and run:
 
     ./vk_loader_validation_tests
+
+## Building on Fuchsia
+
+Fuchsia uses the project's GN build system to integrate with the Fuchsia platform build.
```

Comment:
Should we indicate that Fuchsia support is Alpha since the OS has not yet been released?

Reply:
+1

Reply:
Yes we should.  @lenny-lunarg please go ahead and add the note.